### PR TITLE
Add battery information in the about menu and the ion module

### PR DESCRIPTION
**Pull request by FaustinM**
See #220

### CODE REVIEW

File: python/port/mod/os/modos.cpp [hunk 1]
```diff
@@ -111,5 +111,4 @@ mp_obj_t modos_listdir(void) {
   }
   
   return list;
-}
-
+}
```
Comment (M4xi1m3) [inline python/port/mod/os/modos.cpp:114]: ?

**Review by M4xi1m3 — CHANGES_REQUESTED** [review]
Please put line feeds at the end of your files.

I don't know is this is such a good idea, as the battery voltmeter is very imprecise.

File: python/port/mod/os/modos.cpp (hunk 1 above)
Comment (FaustinM) [reply python/port/mod/os/modos.cpp:114]: Just a mistake, I will revert the modification in the next commit

**Comment by FaustinM** [comment]
I think it can be useful if a python script want to know the current status of the battery. By example, for a benchmark script, to ensure battery is sufficient and prevent the shutdown of the calculator at the end of the script. It's always a good thing to have it 😉 !

**Comment by M4xi1m3** [comment]
Then I think it would be wise to also include a way to get the state (as of the 3 states of battery level ion has) in which the calculator is, as the voltage won't mean anything to some people

**Pull request closed without merging**